libavpipe/src/avpipe_copy_mpegts.c

```diff
@@ -24,7 +24,7 @@ copy_mpegts_set_encoder_options(
     int timebase)
```

Comment:
I assume this is timescale

Reply:
Yes - I added a comment to rename later.  It will be best to have a PR with just non-functional cleanup for this.